### DISCUSSION TITLE
第一次用绳网

### DISCUSSION THREAD

**Discussion by tuhfe3-piWvex-cupjin** [share121/inter-knot#4656] [其他]
感觉有点复杂啊，该怎么玩，有人能帮帮我嘛

**Comment by 927484059** [comment]
如果有委托的话准备好菲林，然后会有人接的

**Comment by tuhfe3-piWvex-cupjin** [comment]
我勒个豆


---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 发送日期 | 2024年09月09日 12:16 |
| 收件人 | share121/inter-knot ***@***.***> |
| 抄送人 | tuhfe3-piWvex-cupjin ***@***.***>,
Author ***@***.***> |
| 主题 | Re: [share121/inter-knot] 第一次用绳网 (Discussion #4656) |

如果有委托的话准备好菲林，然后会有人接的

**Comment by dinkaka** [comment]
楼主有委托吗，我接了

**Comment by mclinyu** [comment]
这委托我接了
